drivers/pcie/host/msi.c

```diff
@@ -278,8 +277,11 @@ bool pcie_msi_enable(pcie_bdf_t bdf,
 			disable_msi(bdf, base);
 		}
 
+		if (base_msix != 0U) {
```

Comment:
refactor this with previous if block as well, so:
```
if (base_msix != 0) {
        if (base != 0) {
                disable_msi();
        }

        base = base_msix;
        msi = false;
}
```

Reply:
fixed this way